core-services/src/main/java/org/zalando/nakadi/service/publishing/EventOwnerExtractorFactory.java

```diff
@@ -44,8 +44,8 @@ public EventOwnerExtractorFactory(final FeatureToggleService featureToggleServic
         return (batchItem) -> {
             try {
                 final JsonPathAccess jsonPath = new JsonPathAccess(batchItem);
-                final String value = jsonPath.get(selector.getValue()).toString();
-                return null == value ? null : new EventOwnerHeader(selector.getName(), value);
+                final Object value = jsonPath.get(selector.getValue());
```

Comment:
is it possible jsonPath.get returns null ?

Reply:
it looks like it is gonna be exception if there is no such path

Reply:
Right, jsonPath.get would throw `null` ideally. However, because we use the self-implemented `StrictJsonParser`, it returns `JsonObject.NULL` for `null` fields.

In case of no path, it would throw exception, and this extractor would return `null`